nixos/modules/programs/sway.nix

```diff
@@ -70,4 +80,6 @@ in
     fonts.enableDefaultFonts = mkDefault true;
     programs.dconf.enable = mkDefault true;
   };
+
+  meta.maintainers = with lib.maintainers; [ gnidorah primeos ];
```

Comment:
@gnidorah Quick question: Would you like to maintain this module? And is it ok if I add myself as well?

Reply:
@primeos Sure, thanks.